src/Frontend/src/app/features/dashboard/components/ai-command-widget/ai-command-widget.component.html

```
@@ -0,0 +1,184 @@
+<article class="rounded-2xl border border-border bg-card p-4 shadow-sm">
+  <header class="mb-4 flex items-center gap-2">
+    <div class="flex h-8 w-8 items-center justify-center rounded-full bg-muted">
+      <lucide-icon name="sparkles" class="h-4 w-4 text-primary"></lucide-icon>
+    </div>
+    <h2 class="text-sm font-semibold text-foreground">{{ title() }}</h2>
+  </header>
+
+  <section class="mb-4 max-h-72 space-y-3 overflow-y-auto rounded-xl border border-border bg-background/80 p-3">
+    @for (message of messages(); track message.id) {
+      <div [class.items-end]="message.role === 'user'" class="flex flex-col gap-1">
+        <div
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)